cdc/log.cc

```diff
@@ -547,6 +547,12 @@ api::timestamp_type find_timestamp(const schema& s, const mutation& m) {
                     [&] (collection_mutation_view_description mview) {
                 t = mview.tomb.timestamp;
                 if (t != api::missing_timestamp) {
+                    // A collection tombstone with timestamp T can be created with:
+                    // UPDATE ks.t USING TIMESTAMP T + 1 SET X = null WHERE ...
+                    // where X is a non-atomic column.
+                    // This is, among others, the reason why we show it in the CDC log
+                    // with cdc$time using timestamp T + 1 instead of T.
+                    t += 1;
```

Comment:
Are you not doing this twice now? split.cc:120?

Reply:
There are two independent code paths, one goes through `find_timestamp` (if we don't split), the other goes through `split`. Duplicate logic, unfortunately.

I wanted to have everything to go through the split logic, but there was an argument about efficiency or something.